### ISSUE TITLE
The NOVA Maven server is down

### ISSUE THREAD

**Issue by ExE-Boss** [NOVA-Team/NOVA-Core#207]
The [maven.novaapi.net](http://maven.novaapi.net) server appears to be down on my end.
I’m unsure if this is just me or if the server is down for everyone.

**Comment by RX14** [comment]
Yeah, the VPS died long ago, I still have backups though I think.

**Comment by ExE-Boss** [comment]
@RX14 Can please you upload them somewhere so that we can download them while the Maven server is down? (GitHub Releases or Dropbox come to mind, other file sharing services should also work)

**Comment by skyem123** [comment]
Well... @RX14 has his own Maven repo/server.

**Comment by RX14** [comment]
I might be able to upload the files tonight. If @calclavia could work with me to redirect dns that would be great.

**Comment by calclavia** [comment]
@RX14 I don't remember owning the domains?

**Comment by RX14** [comment]
oh, i thought stan transferred the domain to you @calclavia

**Comment by drkoaeg** [comment]
Any updates? Would you please get the server run again?

**Comment by RX14** [comment]
I would need DNS to be edited by @stanhebben to solve this.

**Comment by stanhebben** [comment]
Which server should it point to?

**Comment by RX14** [comment]
@stanhebben current.rx14.co.uk

**Comment by stanhebben** [comment]
Although the domain is mine, it seems I delegated control over the DNS to someone else. I can't see whom, I think @Kubuxu , or would you manage it @calclavia ?

**Comment by Kubuxu** [comment]
It is resolved by cloudflare, so @calclavia, I never used CF.

**Comment by calclavia** [comment]
I added a CNAME ci.novaapi.net -> current.rx14.co.uk

**Comment by RX14** [comment]
Wasn't the idea to get maven back online not jenkins?

If you cname maven.novaapi.net I'll get that set up tomorrow.

**Comment by calclavia** [comment]
Done

**Comment by dawgermany** [comment]
Neither artifactory nor jenkins nor javadoc nor maven is online

**Comment by RX14** [comment]
It's a lot of work for me to put in for a dead project. I don't think it's worth it.

**Comment by dawgermany** [comment]
@RX14 so is NOVA dead?

**Comment by asiekierka** [comment]
It's been effectively dead since autumn '15.

2016-08-17 20:20 GMT+02:00 dawgermany <notifications@github.com>:

**Comment by ExE-Boss** [comment]
Someone needs to save NOVA, because it would make updating mods to a new version of Minecraft and any other supported voxel games so much easier.

**Comment by asiekierka** [comment]
Not happening. It was way too ambitious and couldn't be done in hobby time.

**Comment by dawgermany** [comment]
@asiekierka but theres a issue to this where the authors state the opposite though

**Comment by dawgermany** [comment]
also you can't download any of the wrappers anymore and you can't build mods which rely on the nova api anymore (e.g. EDX by calclavia )

**Comment by calclavia** [comment]
If someone else wishes to lead the project, feel free to take over and set up all dependencies.

**Comment by ExE-Boss** [comment]
You could ask around the Minecraft Forge forums, I’m sure that someone from the modding community would be willing to take over NOVA.
I wish that I could do it, but unfortunately, I have responsibilities keeping me from leading NOVA at the present time and for the foreseeable future.

**Comment by ExE-Boss** [comment]
Seeing as NOVA is now being maintained again.
We really should get Maven up and running once more.
